### PR TITLE
add simple HTS221, LSM303 and LSM6DSL examples

### PR DESCRIPTION
**Pull request by raiden00pl**

### CODE REVIEW

File: examples/lsm6dsl_reader/lsm6dsl_reader_main.c [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+      return -ENOENT;
+    }
+
+  ret = ioctl(fileno(sensor), SNIOC_START, 0);
+  if (ret < 0)
+    {
+      printf("IOCTL SNIOC_START failed %d\n", ret);
+    }
+
+  for (i = 0; 1; i++)
```
Comment (Ouss4) [inline examples/lsm6dsl_reader/lsm6dsl_reader_main.c]: Is an infinite loop exactly what's wanted here?

Comment (raiden00pl) [reply examples/lsm6dsl_reader/lsm6dsl_reader_main.c]: Yes, at the moment these are just simple infinite read.

Comment (Ouss4) [reply examples/lsm6dsl_reader/lsm6dsl_reader_main.c]: Will they be changed in the future?
If not, I'll turn them to `for (;  ; ) `  and merge, if you don't mind.

Comment (raiden00pl) [reply examples/lsm6dsl_reader/lsm6dsl_reader_main.c]: I don't have any plans for now. You can change them

**Pull request merged**